### DISCUSSION TITLE
orderin list questinos

### DISCUSSION THREAD

**Discussion by Cogur** [Zer0-bit/gaggiuino#179] [Q&A] [answered]
Hi I'm about to order the list of items 

but I have some questions in advance.

1. Arduino Nano AT328: https://www.aliexpress.com/item/4000903444456.html?spm=a2g0s.9042311.0.0.1b8c4c4dzBsUSa 
in this URL there is no item called exactly like that, is the one called Blue-welded-328 ok?

2. LCD screen: Nextion NX3224F024
3. [Thermo-resistant cables AWG15 and AWG20 ( 1m black/red ) and AWG26 ( 5m black/red/yellow/blue )](https://bit.ly/3tjSQbI)
Do you mean one of each color? like 
- AWG15 1m black 
- AWG15 1m red 
- AWG20 1m black 
- AWG20 1m red 
- AWG26 5m black 
- AWG26 5m red 
- AWG26 5m yellow 
- AWG26 5m blue
because thats a lots of cable.

4. do you think the pneumatic air connectors can handle the pressure and the temperature the Gaggia needs?
On this link, they say the max temp is 60C https://www.aliexpress.com/item/4001338085412.html
and the max pressure is also on the edge.

**Comment by Zer0-bit** [comment, marked as answer]
Yes to all questions.

**Comment by HPiyadasa** [comment]
Hi, 

For the C-M4 screw K-Type thermocouple sensor]- (https://www.aliexpress.com/item/32842805668.html) what hex bolt size should I select (8X9, 7X7.8, 6X6.8)? 

Thanks!

**Reply by Zer0-bit** [reply]
whichever you want the hex has no importance
